src/Framework/TestCase.php

```diff
@@ -104,6 +104,16 @@ abstract class TestCase extends Assert implements SelfDescribing, Test
      */
     protected $preserveGlobalState = true;
 
+    /**
+     * @var string[]
```

Comment:
I have noticed that a few properties have been added which appear to be lists of `string`s.

In some places `@var string[]` is used and in other places `@var array<string>`.

What do you think about using the same notation in all modified places?
```

Reply:
That would be better, I agree. Myself I got used to `string[]` but the newer notitation is more flexible. Will bring the new properties in line with the Psalm checks